virttest/qemu_vm.py

```diff
@@ -896,25 +896,35 @@ def add_memorys(devices, params):
 
             :param devices: VM devices container
             """
-            options = []
+            options, devs = [], []
+            params.setdefault("mem", "512")
             mem_params = params.object_params("mem")
-            if mem_params.get("maxmem"):
+            mem_params.setdefault("automem", "yes")
+            automem = mem_params["automem"] == "yes"
+            if automem:
```

Comment:
it's really a issue for memory in 1G hugepage,  I think we 1G to align memory size can avoid issues you mention in above.